protos/protos/public/modeldb/ExperimentRunService.proto

```diff
@@ -73,6 +73,7 @@ message Observation {
         common.Artifact artifact = 2;
     }
     int64 timestamp = 3;
+    google.protobuf.Value epoch_number = 4;
```

Comment:
Here type is `google.protobuf.Value` so please add validation at log endpoint service Impl to ensure that the user sets the numeric value only.

Reply:
checking it at https://github.com/VertaAI/modeldb/pull/825/files#diff-0de1b97140d6153d08ac22f8ba1d58d0R257 , doing it twice did not make much sense to me

Reply:
Ok. then I agree with your flow.